src/dtos/dimension.ts

```diff
@@ -1,13 +1,14 @@
 import { DimensionInfoDTO } from './dimension-info';
-import { SourceDTO } from './source';
+import { LookupTableDTO } from './lookup-table';
 
 export interface DimensionDTO {
     id: string;
+    dataset_id: string;
     type: string;
-    start_revision_id: string;
-    finish_revision_id?: string;
-    validator?: string;
-    sources?: SourceDTO[];
+    extractor?: object;
+    joinColumn?: string; // <-- Tells you have to join the dimension to the fact_table
+    factTableColumn: string; // <-- Tells you which column in the fact table you're joining to
+    isSliceDimension: boolean;
+    lookupTable?: LookupTableDTO;
```

Comment:
If we're sticking to the convention of DTO props always being snake_case then we'll want to tidy this up at some point but not a blocker here.